Failed tests (12754): TestBuildInfo

The following test appears to have failed:

[#12754](https://circleci.com/gh/cockroachdb/cockroach/12754):

```
I0212 20:45:26.022866 27942 rand.go:86  Random seed: 1316259932852266527
=== RUN   TestBuildInfo
I0212 20:45:26.024911 27942 acceptance/cluster/docker.go:74  ImagePull cockroachdb/docker-spy:20160209-143235 starting
panic: test timed out after 5m0s

goroutine 15 [running]:
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:703 +0x132
created by time.goFunc
	/usr/local/go/src/time/sleep.go:129 +0x3a

goroutine 1 [chan receive]:
testing.RunTests(0xcde238, 0x110c7c0, 0xe, 0xe, 0x12444b6f61bd2601)
	/usr/local/go/src/testing/testing.go:562 +0x8ad
testing.(*M).Run(0xc8200a5ee8, 0xcddaa0)
	/usr/local/go/src/testing/testing.go:494 +0x70
github.com/cockroachdb/cockroach/acceptance.TestMain(0xc8200a5ee8)
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:140 +0x3e
main.main()
	github.com/cockroachdb/cockroach/acceptance/_test/_testmain.go:78 +0x113

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x1128bc0)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:1029 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:610 +0xbc

goroutine 6 [syscall]:
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 8 [chan receive]:
github.com/cockroachdb/cockroach/acceptance.TestMain.func1()
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:131 +0xd8
created by github.com/cockroachdb/cockroach/acceptance.TestMain
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:139 +0x30

goroutine 9 [IO wait]:
net.runtime_pollWait(0x7fe7641b1f88, 0x72, 0xc8200141f0)
	/usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc82019a610, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82019a610, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82019a5b0, 0xc8201a0000, 0x1000, 0x1000, 0x0, 0x7fe7641aa050, 0xc8200141f0)
	/usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc82002c628, 0xc8201a0000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
testing.tRunner(0xc8200847e0, 0x110c7c0)
	/usr/local/go/src/testing/testing.go:456 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:561 +0x86d

goroutine 19 [select, locked to thread]:
runtime.gopark(0xcde8e0, 0xc820024f28, 0xba24b0, 0x6, 0x431b18, 0x2)
	/usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc820024f28, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc820024f28)
	/usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:227 +0x353
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1721 +0x1

goroutine 12 [select]:
net/http.(*persistConn).readLoop(0xc8201562c0)
	/usr/local/go/src/net/http/transport.go:976 +0xac7
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:685 +0xc78

goroutine 13 [select]:
net/http.(*persistConn).writeLoop(0xc8201562c0)
	/usr/local/go/src/net/http/transport.go:1009 +0x40c
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:686 +0xc9d

goroutine 14 [select]:
github.com/docker/engine-api/client/transport/cancellable.Do.func3(0x7fe7641b0dd0, 0xc820015c90, 0xc82014a1d0, 0xc82001b440)
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:78 +0xee
created by github.com/docker/engine-api/client/transport/cancellable.Do
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:84 +0x335
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/build/circle-deps.sh
+++ b/build/circle-deps.sh
@@ -132,6 +132,8 @@ ls -lah "${builder_dir}"
 # "build/builder.sh push" and the new tag value placed here.
 fetch_docker "builder" "20151210-134003"
 
+fetch_docker "docker-spy" "20160209-143235"
+
 if is_shard 0; then
   # Dockerfile at: https://github.com/cockroachdb/postgres-test
   fetch_docker "postgres-test" "20160203-140220"

